README.md

```diff
@@ -2,82 +2,152 @@
 
 [![Download](https://api.bintray.com/packages/palantir/releases/bulldozer/images/download.svg)](https://bintray.com/palantir/releases/bulldozer/_latestVersion) [![Docker Pulls](https://img.shields.io/docker/pulls/palantirtechnologies/bulldozer.svg)](https://hub.docker.com/r/palantirtechnologies/bulldozer/)
 
-bulldozer is a [GitHub App](https://developer.github.com/apps/) that auto-merges
-pull requests (PRs) when all status checks are green and the PR is reviewed.
+`bulldozer` is a [GitHub App](https://developer.github.com/apps/) that
+automatically merges pull requests (PRs) when (and only when) all required
+status checks are successful and required reviews are provided.
+
+Additionally, `bulldozer` can:
```

Comment:
nit: "can be configured too"? it would make line 19 unnecessary

Reply:
Hmm, I see what you mean. L19 was meant to point out that the configuration for the application in general is in a file in the repository, not that you use the file to configure these specific features. I'll move it around a bit.